instrumentation/micrometer/micrometer-1.5/javaagent/build.gradle.kts

```diff
@@ -0,0 +1,21 @@
+plugins {
+  id("otel.javaagent-instrumentation")
```

Comment:
+100

Reply:
I realized that this would make a very useful library instrumentation some time after opening this PR 😅 
I'll split it out in one of the next PRs - once all meters/instruments are implemented.